apps/experiments/models.py

```diff
@@ -451,6 +451,7 @@ class Participant(BaseTeamModel):
     identifier = models.CharField(max_length=320, blank=True)  # max email length
     public_id = models.UUIDField(default=uuid.uuid4, unique=True)
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, null=True, blank=True)
+    platform = models.CharField(max_length=32)
```

Comment:
What about adding `choices` here, similar to [what we do](https://github.com/dimagi/open-chat-studio/blob/7d21824d0336f392228e5ff9d4900640fbabdba7/apps/channels/models.py#L25) for the `ExperimentChannel`?

Reply:
I see from the [issue](https://github.com/dimagi/open-chat-studio/issues/522) that you intended to add a uniqueness constraint as well? An index might also be a good idea

Reply:
> What about adding `choices` here, similar to [what we do](https://github.com/dimagi/open-chat-studio/blob/7d21824d0336f392228e5ff9d4900640fbabdba7/apps/channels/models.py#L25) for the `ExperimentChannel`?

I decided not to do that since I don't think it adds much value and results in more DB migrations every time we add a platform. Perhaps it's a non-issue though. Let me know if you think it's worth it.

> I see from the [issue](https://github.com/dimagi/open-chat-studio/issues/522) that you intended to add a uniqueness constraint as well? An index might also be a good idea

Thanks for the reminder. I wanted to verify that there weren't any duplicates first which I have now done so I'll add the constraint.

Reply:
> I decided not to do that since I don't think it adds much value and results in more DB migrations every time we add a platform. Perhaps it's a non-issue though. Let me know if you think it's worth it.

For me this adds value more for the devs. When you look at the field definition, you immediately see which values are allowed/used. I'm not against it being removed, but am slightly biased toward keeping it. Say a 6/10 (10=keep it, 0=leave it). If you feel stronger than this in the opposite direction, feel free to leave it out :smile:

Reply:
I'm going to leave it off for now since referencing it will require moving a bunch of code around to avoid circular dependencies. I also don't want to copy it otherwise they could get out of sync.